demo/webcam_demo.py

```diff
@@ -0,0 +1,466 @@
+import argparse
+from collections import defaultdict
+
+import cv2
+import mmcv
+import numpy as np
+
+from mmpose.apis import (inference_top_down_pose_model, init_pose_model,
+                         vis_pose_result)
+
+try:
+    from mmdet.apis import inference_detector, init_detector
+    has_mmdet = True
+except (ImportError, ModuleNotFoundError):
+    has_mmdet = False
+
+
+class StopWatch:
```

Comment:
Where does this code come from?

Reply:
It was not adapted from somewhere else.

Reply:
If this util is generally useful, then it can be put into the repo's util

Reply:
moved to mmpose/utils.py